releng/config-forker/main.go

```diff
@@ -351,6 +353,25 @@ func generateNameVariant(name, version string, generic bool) string {
 	return strings.ReplaceAll(name, "-master", suffix)
 }
 
+func generatePresubmitNameVariant(name, version string) string {
+	suffix := "-" + version
+	if !strings.HasSuffix(name, "-master") {
+		return name + suffix
+	}
+	return strings.ReplaceAll(name, "-master", suffix)
+}
+
+func generatePresubmitContextVariant(name, context, version string) string {
+	suffix := "-" + version
+
+	if context != "" {
+		return strings.ReplaceAll(context, "-master", suffix)
+	}
+
+	context = name
+	return strings.ReplaceAll(context, "-master", suffix)
```

Comment:
```suggestion
func generatePresubmitContextVariant(name, context, version string) string {
	suffix := "-" + version

	if context != "" {
		context = name
	}

	return strings.ReplaceAll(context, "-master", suffix)
```
Does that work? The `context = name` line seems like it isn't totally needed if you return `return strings.ReplaceAll(name, "-master", suffix)` or use suggestion. Thanks!

Reply:
> The `context = name` line seems like it isn't totally needed if you return `return strings.ReplaceAll(name, "-master", suffix)` or use suggestion. Thanks!


Definitely looks better. Done. :+1: